.github/workflows/android.yml

```
@@ -16,8 +16,9 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        triplet: ['arm64-android', 'arm-android', 'x64-android', 'x86-android']
-        all_files_access: ['ON', 'OFF']
+        all_files_access: ['OFF']
+        triplet: ['arm64-android', 'arm-neon-android', 'x64-android', 'x86-android']
```

```suggestion
        include:
          - triplet: 'arm64-android'
            qt_arch: 'arm64-android'
          - triplet: 'arm-neon-android'
            qt_arch: 'android_armv7'
          - triplet: 'x64-android'
            qt_arch: 'android_x86_64'
          - triplet: 'x86-android'
            qt_arch: 'android_x86'
```

That should help to dedup

---

@m-kuhn , brilliant.